### PR TITLE
feat: Introducing arm64v8 support for LoRa BasicStation container and LoRaWanNetworkServer

### PR DESCRIPTION
**Pull request by danigian**
# PR for issue #265

## What is being addressed

This PR is introducing Dockerfiles for arm64v8 architectures for both LoRaWan Network Server and BasicsStation

## Validation

- CI passing
- Manually run LBS and LNS containers in a Raspberry Pi 4 (running with 64bit raspbian os)

### CODE REVIEW

File: LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm64v8 [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#This docker builds a container for the LoRa Basics station on arm32 architecture
```
Comment (roel4ez) [inline LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm64v8:1]:
```suggestion
#This docker builds a container for the LoRa Basics station on arm64 architecture
```

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/JoinRequestMessageHandler.cs [hunk 2]
```diff
@@ -266,8 +266,8 @@ internal async Task ProcessJoinRequestAsync(LoRaRequest request)
 
                 var joinAcceptBytes = loRaPayloadJoinAccept.Serialize(appKey);
 
+                // For join accept messages the RX1DROffsetvalue is ignored as the join accept message carry the settings to the device.
```
Comment (roel4ez) [inline LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/JoinRequestMessageHandler.cs:269]: nit: is this relevant?

**Comment by roel4ez** [comment]
Do we need to add something to the docs to explain why there are different architectures and where/how each one should be used?

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/JoinRequestMessageHandler.cs (hunk 2 above)
Comment (danigian) [reply LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/JoinRequestMessageHandler.cs:269]: Without this change I was not able to build locally, therefore I moved the comment before

File: LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm64v8 (hunk 1 above)
Comment (danigian) [reply LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.arm64v8]: Done with a5d0efce

**Comment by danigian** [comment]
> Do we need to add something to the docs to explain why there are different architectures and where/how each one should be used?

Honestly I am not sure. I think a user should be aware of his hw architecture and what to use. Additionally, we have a section in the documentation covering [this](https://azure.github.io/iotedge-lorawan-starterkit/2.1.0/user-guide/devguide/?h=arm32#setup-concentrator-with-azure-iot-edge)

We might want to expand it with arm64v8 option from the dropdown. Wdyt?

**Review by roel4ez — APPROVED** [review]
LGTM

**Pull request merged**